src/components/playback/playbackmanager.js

```diff
@@ -2600,9 +2611,16 @@ class PlaybackManager {
                 // this reference was only needed by sendPlaybackListToPlayer
                 playOptions.items = null;
 
+                autoSetNextTracks(prevSource, mediaStreams, trackOptions, user.Configuration.RememberAudioSelections, user.Configuration.RememberSubtitleSelections);
```

Comment:
Can you elaborate? trackOptions can be empty if there is no prevSource.

Reply:
> Can you elaborate? trackOptions can be empty if there is no prevSource.

I mean the next media source may have `DefaultAudioStreamIndex` and `DefaultSubtitleStreamIndex`. They can be the starting point for `trackOptions`. Then `autoSetNextTracks` overrides them.

But with `!= null`, it will probably work as needed. So never mind.